lib/pry-byebug/cli.rb

```diff
@@ -1 +0,0 @@
-require 'pry-byebug'
```

Comment:
This is only thing I'm worried about. That require was introduce to fix an issue, see here: https://github.com/deivid-rodriguez/pry-byebug/pull/10. But now that I look at it again it doesn't make a lot of sense to me anymore... What do you think?